07_train_final_classifiers/collect_full_models.ipynb

```diff
@@ -0,0 +1,706 @@
+{
```

Comment:
<p>I see that you also like perform some testing between cells. An extra test here would be to manually check one gene and some of its features, and add <code>asserts</code> here, just to make sure nothing got messed up. This is just a suggestion, you don't have to do it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/mpmp/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>